x-pack/test/apm_api_integration/trial/tests/services/top_services.ts

```diff
@@ -56,15 +57,17 @@ export default function ApiTest({ getService }: FtrProviderContext) {
 
           expect(severityScores.filter(Boolean).length).to.be.greaterThan(0);
 
-          expect(severityScores).to.eql([
-            undefined,
-            undefined,
-            undefined,
-            undefined,
-            undefined,
-            'warning',
-            undefined,
-          ]);
+          expectSnapshot(severityScores).toMatchInline(`
+            Array [
+              undefined,
+              undefined,
+              undefined,
+              undefined,
+              undefined,
+              "warning",
+              undefined,
+            ]
+          `);
```

Comment:
Would it make sense to filter out `undefined` severity scores up front:


```suggestion
          const severityScores = response.body.items.map((item: any) => item.severity).filter(Boolean);          
          expect(severityScores.length).to.be.greaterThan(0);
          expectSnapshot(severityScores).toMatchInline(`
            Array [
              "warning",
            ]
          `);
```

Reply:
I think that would make sense, but it might no longer be necessary when we fix https://github.com/elastic/kibana/issues/77083. I've added it as a note there, to be able to merge this in a bit.